app/assets/stylesheets/_forms.scss

```diff
@@ -70,21 +62,17 @@ textarea {
   resize: vertical;
 }
 
-input[type="search"] {
+[type="search"] {
```

Comment:
While the `input` tag isn't required, it is a lot clearer with it in. I'd prefer to keep them unless there's a better reason not to.